api/src/backend/api/specs/v1.yaml

```diff
@@ -3506,6 +3506,27 @@ paths:
         description: A search term.
         schema:
           type: string
+      - in: query
+        name: filter[status]
+        schema:
+          type: string
+          enum:
+          - FAIL
+          - MUTED
```

Comment:
Is this correct? Muted is not a valid finding status, is another attribute.

Reply:
Got it, removed 👍🏼